draft-ietf-git-using-github.md

```diff
@@ -428,6 +447,251 @@ A working group that uses GitHub MAY provide either facility at the request of
+issue tracking might find that distinction sufficient for their needs.
+
+Working Groups or editors might use additional labels as they choose.  Any label
+that is used as part of a process requires that the process be documented and
+announced by Working Group chairs.  Editors SHOULD be permitted to use labels to
+manage issues without any formal process significance being attached to those
+issues.
+
+
+### Design/Editorial Labelling {#label-design}
+
+The most important distinction about an issue is whether it is substantive.  The
+labels of `design` and `editorial` are used to represent this distinction.
+
+An issue labeled as `design` has or might have a substantive effect on a
+document.  For protocol specifications, a `design` issue is one that might
+affect implementations or interoperability requirements.  Addressing a `design`
+issue ultimately requires Working Group consensus, even if the resolution is to
+make no change.
+
+An issue labeled as `editorial` has no substantive effect on a document, except
+to the extent that addressing the issue might make understanding the
+specification easier.  Resolution of `editorial` issues can be left to the
+discretion of editors.
+
+This distinction can be applied to all types of document.  For instance, a
+`design` issue for an Informational document might be raised to discuss
+possible changes to important concepts in the document.
+
+
+### Decision Labelling {#label-decision}
+
+Labels can be used to manage processes.  As documents mature and issues become
+more numerous, labels can be used to clearly mark the status of issues.  In
+particular, labelling of issues can be used to help in managing Working Group
+decisions.
+
+For documents that are less mature, issues with resolutions but no specific
+proposals for changes to text might be marked `editor-ready` as a way of
+signaling that there is consensus about an approach, but no specific proposal.
+Chairs might use this to signal that discussion is complete and that editors are
+to be given discretion in the construction of text.
+
+In contrast, if specific text is a prerequisite for resolving issues, as might
+be the case for more mature documents, a `proposal-ready` label might be used by
+editors to mark issues that they believe to have acceptable resolutions.
+
+For resolved issues, a `has-consensus` label might be used by chairs to mark
+issues for which formal Working Group decisions have been made (Section 6.1 of
+{{!RFC2418}}).
+
+A `v2` or `next-version` label might be used to mark and thereby save issues for
+a future version of or extension to a protocol, particularly where a resolution
+is made to take no action.
+
+
+### Component Labelling
+
+Repositories with multiple interrelated documents or a complex document with
+multiple logical components might benefit from labels that identify different
+aspects of the work.  The choice of appropriate labels for components will
+depend on the structure of specific documents.
```

Comment:
We might want to suggest that labeling of this sort be at the editors' discretion.

Reply:
@MikeBishop, could you file an issue or (better) create a PR?